quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -93,7 +94,7 @@ impl Default for QuickwitNumericOptions {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, utoipa::ToSchema)]
```

Comment:
I believe only the `QuickwitJsonOptions` is useful. We expose `FieldMappingEntryForSerialization` which is not very useful and the name is not great.